sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/support/GetHashMac.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+/*
+ * Disclaimer:
+ *      This class is copied from https://github.com/Microsoft/azure-tools-for-java/ with minor modification (fixing
+ *      static analysis error).
+ *      Location in the repo: /Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/GetHashMac.java
+ */
```

Comment:
Copyright content should be validated by CELA. Please reach out to CELA for this. @anuchandy can provide pointers.

Reply:
Since it's from Microsoft GitHub Org to Azure GitHub Org, there shouldn't be an issue. Both are under MIT as well. @saragluna, I sent the details offline.